components/Layout.tsx

```diff
@@ -1,16 +1,31 @@
 import React from 'react'
-import LandingNav from './LandingNav'
+import AppNav from './AppNavBar'
 import Footer from './Footer'
 
 type Props = {
   children: React.ReactElement
+  loggedIn: boolean
+  username?: string
+  firstName?: string
+  lastName?: string
 }
 
-const Layout: React.FC<Props> = props => {
+const Layout: React.FC<Props> = ({
+  children,
+  loggedIn,
+  username,
+  firstName,
+  lastName
+}) => {
   return (
     <>
-      <LandingNav />
-      <div className="container">{props.children}</div>
+      <AppNav
+        loggedIn={loggedIn}
+        username={username}
+        firstName={firstName}
+        lastName={lastName}
+      />
```

Comment:
We might `useContext()` to pass `Auth data` to the <NavBar /> but maybe it's over-engineering